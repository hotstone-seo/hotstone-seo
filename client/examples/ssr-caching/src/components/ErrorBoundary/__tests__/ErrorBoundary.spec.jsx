import React, { PureComponent } from 'react';
import ErrorBoundary from '../';
import ErrorView from '../ErrorView';

const TestError = () => <div>test</div>;

test('Should render ErrorBoundary correctly', () => {
  const tree = enzyme.mount(
    <ErrorBoundary render={(error, info) => <ErrorView error={error} info={info} />}>
      <TestError />
    </ErrorBoundary>
  );

  expect(tree).toMatchSnapshot();
});
