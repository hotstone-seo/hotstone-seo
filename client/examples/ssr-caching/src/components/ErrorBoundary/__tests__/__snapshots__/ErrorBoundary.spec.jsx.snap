// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ErrorBoundary correctly 1`] = `
<ErrorBoundary
  debug={false}
  errorMessage=""
  render={[Function]}
>
  <TestError>
    <div>
      test
    </div>
  </TestError>
</ErrorBoundary>
`;
